### PR TITLE
op-guide: add notes on mounting certs directory

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/pull/7399.

/cc @xiang90 @heyitsanthony @fanminshi

### CODE REVIEW

**Comment by xiang90** [comment]
LGTM

File: Documentation/op-guide/container.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ sudo rkt trust --prefix coreos.com/etcd
 # gpg key fingerprint is: 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E
 ```
 
-Run the `v3.0.6` version of etcd or specify another release version.
+Run the `v3.1.12` version of etcd or specify another release version.
```
Comment (heyitsanthony) [inline Documentation/op-guide/container.md]: should these just use `latest` so they don't go out of date?

Run the `latest` version of etcd or specify a specific release (e.g., `v3.1.12`).
?

File: Documentation/op-guide/container.md [hunk 2]
```diff
@@ -132,3 +132,31 @@ docker exec etcd /bin/sh -c "export ETCDCTL_API=3 && /usr/local/bin/etcdctl put
 
 To provision a 3 node etcd cluster on bare-metal, you might find the examples in the [baremetal repo](https://github.com/coreos/coreos-baremetal/tree/master/examples) useful.
 
+## Mount certs volume
```
Comment (heyitsanthony) [inline Documentation/op-guide/container.md]: `Mounting a certificate volume`?

File: Documentation/op-guide/container.md [hunk 3]
```diff
@@ -132,3 +132,31 @@ docker exec etcd /bin/sh -c "export ETCDCTL_API=3 && /usr/local/bin/etcdctl put
...
+
+Note that etcd release container does not include default certifications.
```
Comment (heyitsanthony) [inline Documentation/op-guide/container.md]: these should be a single line per the style guide
```
The etcd release container does not include default root certificates. To use HTTPS with certificates trusted by a root authority (e.g., for discovery), mount a certificate directory into the etcd container:
```

**Comment by gyuho** [comment]
@heyitsanthony @fanminshi `coreos.com/etcd:latest` does not work for `rkt`, so just used `v3.1.2`. PTAL.

**Comment by heyitsanthony** [comment]
lgtm

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**